### PR TITLE
UCS-99 : New framework for getter/setter tests at package level

### PR DESCRIPTION
**Pull request by tezcan10**

### CODE REVIEW

File: UCSLibraries/pom.xml [hunk 1]
```diff
@@ -83,6 +83,19 @@
 			<artifactId>wiremock</artifactId>
 			<version>2.18.0</version>
 		</dependency>
+		<dependency>
+		  <groupId>pl.pojo</groupId>
+		  <artifactId>pojo-tester</artifactId>
+		  <version>0.7.6</version>
+          <scope>test</scope>
```
Comment (hexec) [inline UCSLibraries/pom.xml]: shouldn't the IDE be doing the automatic indentation on save?

File: UCSLibraries/src/test/java/it/cnr/iit/AbstractPojoTest.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+import org.junit.Test;
+
+import pl.pojo.tester.api.DefaultPackageFilter;
+import pl.pojo.tester.api.assertion.Method;
+
+public class AbstractPojoTest {
+
+    @Test
+    public void Should_Pass_All_Pojo_Tests() {
```
Comment (hexec) [inline UCSLibraries/src/test/java/it/cnr/iit/AbstractPojoTest.java]: first letter is capital? maybe sonar will complain about this => should_Pass_All_Pojo_Tests

Comment (tezcan10) [reply UCSLibraries/src/test/java/it/cnr/iit/AbstractPojoTest.java]: well spotted :-)
done

File: UCSLibraries/pom.xml (hunk 1 above)
Comment (tezcan10) [reply UCSLibraries/pom.xml]: I think not. Only java classes gets formatted

**Pull request merged**